src/app/controllers/encrypt-submissions.server.controller.js

```diff
@@ -202,15 +202,25 @@ exports.saveResponseToDb = function (req, res, next) {
  */
 exports.getMetadata = function (req, res) {
   let pageSize = 10
-  let { page } = req.query || {}
+  let { page, filterBySubmissionRefId } = req.query || {}
```

Comment:
if i could make a small nitpick here, which is to rename this query parameter to `submissionId` instead?

Reply:
Cool, will rename.